package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "local-home-virual-environment",
```

Comment:
add author?

Reply:
Google LLC

Reply:
Added `"author": "Google"` to match the Local Home SDK and SDK sample.